### DISCUSSION TITLE
Admin auth token expiration

### DISCUSSION THREAD

**Discussion by nCubed** [keygen-sh/community#49] [Q&A] [answered]
From the docs [Generate a token](https://keygen.sh/docs/api/tokens/#tokens-generate):

> By default, user tokens expire in 2 weeks, and admin tokens do not expire. 

Does this policy also apply for the admin token generated in the Keygen admin panel? Settings > Current admin token

Or does the admin token provided in the Keygen admin panel expire after some time and/or logging out?

**Comment by nCubed** [comment, marked as answer]
Answered my own question. The admin token generated in the Keygen admin panel expires as soon as you log out of the Keygen admin panel.

**Reply by ezekg** [reply]
Correct. Logging out will immediately revoke the token. By default, admin tokens do not expire.
